src/main/java/DukeBody/Duke.java

```diff
@@ -1,3 +1,6 @@
+package DukeBody;
```

Comment:
I think it was a good idea to separate the project into task and body packages, but according to the module's coding standard, names representing packages should all be in lower case